packages/env/index.js

```diff
@@ -1,5 +1,20 @@
 const { EnvironmentPlugin } = require('webpack');
 
-module.exports = ({ config }, envs = []) => config
-  .plugin('env')
-  .use(EnvironmentPlugin, ['NODE_ENV', ...envs]);
+module.exports = ({ config }, envs = []) => {
+  let pluginOptions;
+
+  if (Array.isArray(envs)) {
+    pluginOptions = ['NODE_ENV', ...envs];
+  }
+  else {
+    pluginOptions = [
```

Comment:
> @ftes i think it should be backward compatible (and my PR reflects it: 4f6b4bf).

Your PR is slightly different because it wraps the object inside an array.

I think this is what @eliperelman meant with
>Unfortunately the else clause will violate that.